modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/type/EpisodeViewSource.kt

```diff
@@ -0,0 +1,19 @@
+package au.com.shiftyjelly.pocketcasts.models.type
+
+enum class EpisodeViewSource(val value: String) {
+    DISCOVER("discover"),
```

Comment:
I added this class to `services.models` module. Adding it in the `feature.podcasts` module where the `EpisodeFragment` resides made it inaccessible from the `services.ui` module's `openEpisodeDialog(...)` due to cyclic dependency.

Other places I can think of adding it are the settings or the analytics modules. Or I can add the source as a string in `openEpisodeDialog(...)` but we'll lose type-safety in this case.

Let me know if you can think of a better home for the class.

Reply:
Adding this to `models` like you have is probably what I would have done too.

> ...Or I can add the source as a string in openEpisodeDialog(...) but we'll lose type-safety in this case.

Yeah, I wouldn't want to lose the type-safety.